plugin-veracode-sast/pom.xml

```diff
@@ -39,6 +39,12 @@
 			<artifactId>synapse-core</artifactId>
 			<version>${project.version}</version>
 		</dependency>
+		
```

Comment:
Can we delete the `vosp-api-wrappers` above since we're pulling in the lib now?

Reply:
yes. done